crates/proof-of-sql/src/base/commitment/column_commitment_metadata.rs

```diff
@@ -725,6 +798,10 @@ mod tests {
             column_type: ColumnType::Scalar,
             bounds: ColumnBounds::NoOrder,
         };
+        let tinyint_metadata = ColumnCommitmentMetadata {
```

Comment:
`TinyInt` should not be in a test named `we_can_difference_smallint_matching_metadata`

Reply:
but its not in `we_can_difference_smallint_matching_metadata`

it's in `we_cannot_perform_arithmetic_on_mismatched_metadata`.